frontend/src/components/session/session_form.jsx

```diff
@@ -0,0 +1,138 @@
+import React from "react";
+import { Link } from "react-router-dom";
+import '../reset.css';
+import './session_form.css';
+
+class SessionForm extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      firstName: '',
+      lastName: '',
+      email: '',
+      password: '',
+      zipCode: ''
+    };
+    this.handleSubmit = this.handleSubmit.bind(this);
+  }
+
+  componentWillUnmount() {
+    this.props.clearErrors();
+  }
+
+  handleSubmit(e) {
+    e.preventDefault();
+    this.props.processForm(this.state);
+  }
+
+  update(field) {
+    return e => this.setState({ [field]: e.target.value });
+  }
+
+  render() {
+    const { errors, formType } = this.props;
+    const isSignup = formType === "signup";
+    const header = (
```

Comment:
their SessionForm has a very simplified version of the HeaderNav on top, with only the logo in the middle. so we either need to make the HeaderNav adjust to "/login" and "/signup" routes or do something specific for the form (recycling the logo).